fcs/build.fsx

```diff
@@ -14,7 +14,9 @@ open Fake.ReleaseNotesHelper
 // prevent incorrect output encoding (e.g. https://github.com/fsharp/FAKE/issues/1196)
 System.Console.OutputEncoding <- System.Text.Encoding.UTF8
 
-File.WriteAllText(__SOURCE_DIRECTORY__ + "/../tests/TestResults/notestsyet.txt","No tests yet")
+CleanDir (__SOURCE_DIRECTORY__ + "/../tests/TestResults") 
+File.WriteAllText(__SOURCE_DIRECTORY__ + "/../tests/TestResults/notestsyet.txt","No t__SOURCE_DIRECTORY__ + "/../tests/TestResults/notestsyet.txt","No tests yet")
+#endifests yet")
```

Comment:
is this line correct?
+#endifests yet") 

looks odd

Reply:
Yes merge errors